geps/gep-2198.md

```diff
@@ -0,0 +1,103 @@
+# JWT Policy
+
+## TLDR
+
+This document is not intended to describe how a proxy implements JWT handling behavior, there is an RFC that addresses that.
+Rather, this document is to form a common understanding or JWT use cases as it relates to configuring and handling JWTs as par of a workflow as it relates to a Gateway and establish a framework from which to build the common parts of the Policy.
+
+## Goals
+
+- Define how JWT is used to form a common understanding and establish a common language
+- Identify unique use cases for JWT with proxies
+- Propose some properties to form the basis for discussion
+
+## Non-Goals
+
+- Define how proxies implement the JWT standards
+
+## What is a JWT
+
+A [JSON web token](https://www.rfc-editor.org/rfc/rfc7519) is a common pattern of presenting authorization data (claims) between a front end or client system and a back end.
+A JWT by itself it is only one piece of the [authentication triad](https://csrc.nist.gov/glossary/term/Multi_Factor_Authentication#:~:text=The%20three%20authentication%20factors), which is "something you have".  We will assume that some other process satisfied "something you are" and "something you know".  And thus the JWT is generally considered the result of some type of authentication or formation of a trust.
+
+JWT's can be encrypted, have methods for validation, can contain a variety of claims, and be used in gate keeping (validation) and authorization (claims matching) workflows.
```

Comment:
Maybe we can also mention JWE.

Reply:
Does JWE need to be called out specifically?
Since it is part of the JWT spec.  But the Policy does not / should not care - IMHO

Reply:
I see.